<!DOCTYPE HTML>
<html>

	<head>
		<title>Dahlquist Lab - Research</title>
	    <meta charset="UTF-8">
	    <link rel="stylesheet" href="../assets/css/reset.css" type="text/css">
	    <link rel="stylesheet" href="../assets/css/main.css" type="text/css">
	</head>

    <body>

        <div class="titlebar">
			<a href="http://www.lmu.edu" target="_blank">
				<img src="../assets/images/LMULogoHQ2.png" alt="LMU logo" width="200" height="118" id="logo" />
			</a>
			<p id="title">
				<a href="../">Dahlquist Lab</a>
			</p>
        </div>

        <div class="sidebar">
			<div id="sidebarMenu">
                    <ul>
                        <li>
                            <a href="../"> Home </a>
                        </li>
                        <li>
                            <a href="http://xmlpipedb.cs.lmu.edu" target="_blank">XMLPipeDB</a>
                        </li>
                        <li>
                            <a href="http://dondi.github.io/GRNsight/" target="_blank">GRNsight</a>
                        </li>
                        <li>
                            <a href="http://kdahlquist.github.io/GRNmap/" target="_blank">GRNmap</a>
                        </li>
                        <li>
                            <a href="../research/">Research</a>
                        </li>
                        <li>
                            <a href="../courses/">Courses</a>
                        </li>
                        <li>
                            <a href="../people/">People</a>
                        </li>
                        <li>
                            <a href="../publications/">Publications</a>
                        </li>
                        <li>
                            <a href="../women/">Women in Science</a>
                        </li>
                        <li>
                            <a href="http://cse.lmu.edu/department/biology/">Biology Department</a>
                        </li>
                        <li>
                            <a href="../links/">Links</a>
                        </li>
                        <li>
                            <a href="../contact/">Contact Us</a>
                        </li>
                    </ul>
			</div>
        </div>
        
        <div class="content"> 
            <p id="pageTitle">Research</p>
            <br /> 
            <div id="pageText">
                <p>
                    The Dahlquist Lab performes three distinct, but related research projects. The common thread amongst these projects is that they employ the techniques of bioinformatics and genomics and the perspective of systems biology. All of these projects involve undergraduate or Master's level students and most involve interdisciplinary collaborations with other faculty at LMU (see: <a href="../people">People</a>). These research projects have also been brought into the classroom (see: <a href="../courses">Courses</a>). For lab protocols see the <a href="http://www.openwetware.org/wiki/Dahlquist" target="_blank">Dahlquist Lab site</a> at <a href="http://www.openwetware.org/wiki/Main_Page" target="_blank">OpenWetware.org</a>.
                </p>
                <br />
                <br />
                <p class="researchTitle">
                    Creation of MAPPs and Gene Databases for the GenMAPP software suite; XMLPipeDB: A Reusable, Open Source Tool Chain for Building Relational Databases from XML Sources (<a href="http://xmlpipedb.cs.lmu.edu/" target="_blank">XMLPipeDB Web Site</a>)
                </p>
                <br />
                <p> 
                    <a href="http://www.genmapp.org" target="_blank">GenMAPP</a> (Gene Map Annotator and Pathway Profiler) is a tool for viewing and analyzing DNA microarray and other types of high-throughput data on "MAPPs" representing biological pathways or other functional groupings of genes. GenMAPP has graphics tools for drawing MAPPs, but also has an underlying Gene Database which allows users to give the genes placed on the MAPP an identifier from a public database. This feature makes it possible for users to import gene expression data into GenMAPP and color-code the genes according to the data. MAPPFinder works with GenMAPP to determine which Gene Ontology terms are over-represented among genes changed in the imported expression data. The Dahlquist Lab has been working to address two limitations of GenMAPP: (1) GenMAPP is only as useful as the number of MAPPs available; and (2) GenMAPP can only be used with species for which a GenMAPP-formatted Gene Database is available. Vassar College students, Meredith Braymer '04 and Jessica Heckman '05, built a complete set of 120 metabolic pathway MAPPs for yeast, based on the pathway information available from the <span class="italics">Saccharomyces</span> Genome Database (SGD). These MAPPs have been made available for download from the <a href="http://www.genmapp.org" target="_blank">GenMAPP.org</a> web site for use by the entire research community.
                </p>
                <br />
                <p>
                    Currently, the main GenMAPP development group only supports Gene Databases for eleven species, none of which are bacteria or plants, because they use Ensembl as the main data source. To expand the number of species available for analysis in GenMAPP and to make our system more robust to changes in source data formats, my collaborator we have created XMLPipeDB, a reusable, open source tool chain for automatically building relational databases from an XML schema (XSD). "XML" stands for "eXtensible Mark-up Language." There is a growing trend in the bioinformatics field to provide data in XML format because an accompanying XSD or DTD document gives a complete description of the format of the XML data that is less idiosyncratic than other formats and can be easily read programmatically. While XMLPipeDB is a general-purpose tool that can be used for any type of XML data, we have used it to solve the Gene Database creation problems faced by the main GenMAPP group. The software uses UniProt as the main data source, opening up the possibility of creating Gene Databases for hundreds of species, is robust to changes in source file formats, uses XML sources wherever possible, takes advantage of existing open source tools, and limits the manual manipulation of the data. The XMLPipeDB software suite includes the individual programs, XSD-to-DB, XMLPipeDB Utilities, and GenMAPP Builder. XSD-to-DB reads an XSD (XML schema) and automatically generates the SQL schema file, Java classes, and Hibernate mappings needed to create a relational database. XMLPipeDB Utilities is a general purpose library for performing simple database functions such as importing XML data into the relational database and running simple queries. GenMAPP Builder is a downstream application that exports the data as a GenMAPP-formatted Gene Database. GenMAPP Builder has been used to create a Gene Database for <span class="italics">Escherichia coli</span> K12 (available from <a href="http://www.genmapp.org" target="_blank">GenMAPP.org</a> and <a href="http://sourceforge.net/projects/xmlpipedb/files/E.%20coli%20K12%20Gene%20Database/" target="_blank">SourceForge.net</a>) and for <span class="scienfiticName">Arabidopsis thaliana </span>(beta version available from <a href="http://sourceforge.net/projects/xmlpipedb/files/A.%20thaliana%20Gene%20Database/" target="_blank">SourceForge.net</a>). 
                </p>
                <br />
                <p>
                    XMLPipeDB was created by the Bioinformatics Group at Loyola Marymount University, a collaborative research group founded by Dr. Kam D. Dahlquist and Dr. John David N. Dionisio (Department of Electrical Engineering &amp; Computer Science). XMLPipeDB development began in Spring 2006 as a group project in a special studies course in Bioinformatics (<a href="http://myweb.lmu.edu/dondi/spring2006/cmsi698/" target="_blank">CMSI 698/BIOL 498</a>) team-taught by Drs. Dahlquist and Dionisio. Development continued in a second special studies course, Open Source Software Development Workshop (<a href="http://myweb.lmu.edu/dondi/summer2006-1/cmsi698/index.html" target="_blank">CMSI 698/598</a>) and is ongoing. XMLPipeDB also follows the philosophy of Dr. Dionisio&rsquo;s <a href="http://myweb.lmu.edu/dondi/research/#Recourse" target="_blank">Recourse</a> project, which seeks to introduce an open source culture into the computer science curriculum. XMLPipeDB is available under the GNU Lesser General Public License (LGPL); software packages, documentation, and Gene Databases can be downloaded from the project site at <a href="http://sourceforge.net/project/showfiles.php?group_id=158533" target="_blank">SourceForge.net</a>.</li>
                </p>
                <br />
                <br />
                <p class="researchTitle">
                    The Global Transcriptional Response of <span class="italics">Saccharomyces cerevisiae</span> to Cold Shock and Recovery
                </p>
                <br />
                <p>
                    The complete sequencing of the human genome and those of other major model organisms, along with the invention of high-throughput methods to measure gene expression has propelled biology into the genomics era. Where once biologists could only study genes one at a time, DNA microarrays now generate data routinely for thousands of genes in a single experiment. My research harnesses the power of DNA microarray and other types of genomic data to elucidate the systems level properties of gene regulatory networks in the budding yeast, <span class="italics">Saccharomyces cerevisiae</span>.  Yeast responds to environmental stresses through characteristic programs of gene expression. The transcriptional response to heat shock and a variety of other stressors such as changes in nutrient availability, osmolarity, and oxidative stress have been well characterized. However, the response to cold shock has been less well studied. Previous studies on the transcriptional response of budding yeast to cold shock have revealed that the response can be divided into a set of early response genes (after 15 minutes to 2 hours of cold temperatures) and late response genes (after 12 to 60 hours of cold temperatures) (Sahara <span class="italics">et al</span>., 2002<span class="italics"> J Biol Chem</span> 277:50015; Kandror <span class="italics">et al</span>., 2004<span class="italics"> Mol Cell</span> 13:771; Schade <span class="italics">et al</span>., 2004<span class="italics"> Mol Biol Cell</span> 15:5492). The late response genes include the ESR genes induced by many environmental stresses and are regulated by the Msn2p/Msn4p transcription factors (Schade <span class="italics">et al</span>., 2004). While these studies have begun to characterize the transcriptional response of yeast to cold shock, they have not determined which transcription factors were responsible for the induction of the early response genes, the genes uniquely responding to cold shock. 
                </p>
                <br />
                <p>
                    The global transcriptional response to the environmental stress of cold shock and subsequent recovery in <span class="italics">Saccharomyces cerevisiae</span> has been measured using DNA microarrays. DNA microarrays measure the mRNA levels of all 6000 genes in yeast simultaneously, giving a snapshot of all transcriptional activity in the cells at one time. DNA microarrays were obtained through the <a href="http://www.bio.davidson.edu/projects/gcat/gcat.html" target="_blank">Genome Consortium for Active Teaching</a>. DNA microarrays obtained in this manner are for the express use of undergraduates. Vassar College students, Meredith Braymer &rsquo;04 and Philopose Mulugeta &rsquo;07 and LMU students, Heather King &rsquo;06, Matthew Mejia &rsquo;07, Wesley Citti &rsquo;08,  Robert Hybki &rsquo;08, Elizabeth Liu &rsquo;08, Olivia Sakhon &rsquo;08, Kevin Entzminger &rsquo;09, Stephanie Kuelbs &rsquo;09, and Kenny Rodriguez &rsquo;'09 have all performed DNA microarray experiments related to this project. Wild type yeast cells (BY4741) were grown to early log phase at 30&deg;C, then shifted to 13&deg;C for 60 minutes, and then shifted back to 30&deg;C for another 60 minutes. Samples were collected before cold shock (t 0), after 15, 30, and 60 minutes of cold shock (t 15, t 30, and t 60), and after 30 and 60 minutes (t 90, t 120) of recovery at 30&deg;C. Yeast samples were collected for four independent replicates of the time course experiment. Total RNA was purified from each of the samples. The mRNA was amplified, labeled with Cy3 and Cy5 dyes by the indirect method, and hybridized to DNA microarrays according to the manufacturers&rsquo; protocols provided with the reagents (See also the <a href="http://www.openwetware.org/wiki/Dahlquist" target="_blank">Dahlquist Lab site</a> at OpenWetware.org for protcols). These data are currently being analyzed. Click <a href="microarraygallery/">here</a> to view some pictures of DNA microarrays hybridized by students in my lab. 
                </p>
                <br />
                <br />
                <p class="researchTitle">
                    Mathematical Modeling of the Transcriptional Network Controlling the Environmental Stress Response in <span class="italics">Saccharomyces cerevisiae</span>
                </p>
                <br />
                <p>
                    Gene expression is a complex biological process in which cells first transcribe their genes encoded in the DNA into an intermediary known as mRNA. Then the cell translates the mRNAs into proteins. Transcription factors are regulatory proteins which increase or decrease the rate at which a cell transcribes a gene. Recently, genome-wide location analysis has determined the relationships between transcription factors and their target genes on a global scale in budding yeast, <span class="italics">Saccharomyces cerevisiae </span>(Lee <span class="italics">et al</span>., 2002<span class="italics"> Science</span> 298:799; Harbison <span class="italics">et al</span>., 2004 <span class="italics">Nature</span> 431:99). While these data have identified properties of the network topology, they do not reveal the dynamics of the behavior of the network. Using differential equations, we have modeled how the concentrations of proteins in the cell change over time for a subset of a real gene expression network of twenty-one transcription factors controlling the environmental stress response in yeast. The differential equations governing the rate of change of concentration for each protein in the network were based on a sigmoidal function. A weight parameter determines how each transcription factor affects the transcriptional and translational rate of its target gene. The weights were optimized to experimentally derived gene expression data from yeast exposed to the environmental stress of cold shock. Sensitivity analysis was performed to understand the behavior of the different parameters in the model. We then used the model to generate a simulated gene expression dataset giving the steady-state concentrations of each protein after a period of time has elapsed. The simulated data determined which transcription factors have a greater impact on the overall dynamics of the network. Then each gene in the network was systematically deleted <span class="italics">in silico</span> to determine how the steady-state concentrations of the proteins in the network changed after the deletions. Nathan Wanner '07 began this project in collaboration with Dr. Erika Camacho (who is now at Arizona State Unversity) and won third prize at the <a href="http://www.sdcsb.org/index.php?p=news&op=view&id=16" target="_blank">San Diego Consortium for Systems Biology Symposium: Systems to Synthesis</a>, held at the Salk Institute on January 19, 2007. Currently this work is being performed by Stephanie Kuelbs '09 in collaboration with Dr. Ben Fitzpatrick (LMU Mathematics) as part of the NSF-Interdisciplinary Training for Undergraduates in Biological and Mathematical Sciences (UBM) project at LMU: <a href="http://myweb.lmu.edu/bfitzpatrick/biomath/ASBS.htm" target="_blank">Analysis of Stress in Biological Systems</a>.
                </p>
                <br />
                <br />
                <p id="lastModified">Last modified: 9/3/15</p>
            </div>
            <div class="footer">
				<hr id="separatingLine">
				<div id="footerMenu">
                    <ul>
                        <li>
                            <a href="../"> Home </a>
                        </li>
                        <li>
                            <a href="http://xmlpipedb.cs.lmu.edu" target="_blank">XMLPipeDB</a>
                        </li>
                        <li>
                            <a href="http://dondi.github.io/GRNsight/" target="_blank">GRNsight</a>
                        </li>
                        <li>
                            <a href="http://kdahlquist.github.io/GRNmap/" target="_blank">GRNmap</a>
                        </li>
                        <li>
                            <a href="../research/">Research</a>
                        </li>
                        <li>
                            <a href="../courses/">Courses</a>
                        </li>
                        <li>
                            <a href="../people/">People</a>
                        </li>
                        <li>
                            <a href="../publications/">Publications</a>
                        </li>
                        <li>
                            <a href="../women/">Women in Science</a>
                        </li>
                        <li>
                            <a href="http://cse.lmu.edu/department/biology/">Biology Department</a>
                        </li>
                        <li>
                            <a href="../links/">Links</a>
                        </li>
                        <li>
                            <a href="../contact/">Contact Us</a>
                        </li>
                    </ul>

				    <p id="disclaimer">
				        This web page is not an official Loyola Marymount University web page. Comments on this page should be directed to kdahlquist[at]lmu[dot]edu.
				    </p>
				</div>
            </div>
        </div>

		<script src="../assets/js/jquery-1.11.3.min.js"></script>
		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
		<script src="../assets/js/main.js"></script>

    </body>
    
</html>
